source/includes/pipeline_config/_003-materials.md.erb

```diff
@@ -0,0 +1,9 @@
+<%=
+    describe_sub_object 'The pipeline material object' do
+      type            'String',      'The type of a material.', one_of: %w(git svn hg p4 tfs dependency package plugin)
+      attributes      'Object',      'The [attributes](#the-git-material-attributes) for each material type.'
```

Comment:
I *think* i know why you did that — it is possible to add a quick `<a id='config-git-material-attributes'></a>` just before you describe the material attributes for a quick hyperlink.